LUIS/Runtime SDK/Microsoft.Azure.CognitiveServices.LUIS.Sample/appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "LUIS.Region": "",
```

Comment:
I recommend we get rid of Region, instead, just use Endpoint string.
Regions has brought us a lot of issue in SDK, in the latest SDK we are trying to use Endpoint instead of Region.

"User should copy the Endpoint address from the Azure Portal directly and drop it in to their code without modification".

Reply:
What does the address which the user gets from the Azure Portal look like?

Reply:
@mohabghanem something like "https://westus.api.cognitive.microsoft.com/luis/v2.0"
![image](https://user-images.githubusercontent.com/6030719/42959163-a52ea3a4-8b3c-11e8-9f8d-f328afa75c50.png)

Reply:
But this will not work for the Authoring APIs, they follow the pattern: 
https://westus.api.cognitive.microsoft.com/luis/api/v2.0
A workaround is to do string manipulation to convert the endpoint to the authoring, but I don't think it is a good idea, any suggestions?

Reply:
Why not let the user declare just the region they are in, then add it as a prefix to the urls accordingly?

Reply:
Hmm this is indeed a problem...
Sample code should not be blocked by this though, you can make your own decision in here.
I will start an internal thread to discuss about this.

About "why not just region"... mainly because very soon, the hostname will no longer be always [region].api.cognitive.microsoft.com.

Reply:
ok I will manipulate the string to make the sample work for now